config/locales/user_mailer/en.yml

```diff
@@ -31,7 +31,7 @@ en:
     account_reset_request:
       cancel: Don’t want to delete your account? Sign in to your %{app_name} account
         to cancel.
-      header: Your account will be deleted in 24 hours
+      header: Your account will be deleted in %{interval}
       intro_html: 'As a security measure, %{app_name} requires a two-step process to
         delete your account:<br><br> Step One: There is a 24 hour waiting period
```

Comment:
Do we want to update this "24 hour" as well?

Reply:
We do. Now that I think about it, in the `intro_html`, we might just want to interpolate the number instead of `24 hour` because we don't want to say "There is a 24 hours waiting period"

Reply:
That's a good point. It's not ideal to force ourselves to a specific unit of time (e.g. what if we wanted to extend the time period to 30 days), but I'm not sure there's a better solution aside from changing the content.